scripts/gnosis-addresses.js

```diff
@@ -8,7 +8,7 @@ var deployedContracts = {
 
   "GnosisSafeProxyFactory": {
     "tx": "0x332dad8b0490350e0e342664513f2442a91be1632c03604da9c2a2c30aa70553",
-    "deployedAt": "0xE89ce3bcD35bA068A9F9d906896D3d03Ad5C30EC",
+    "deployedAt": "0xAADFe7925b0Cad895665aDE74f5848043B8c4b7D",
```

Comment:
given I got this addresses when deploying with a fresh node, maybe this is the right address?